src/lib.rs

```diff
@@ -234,10 +240,11 @@ mod tests {
   }
 
   #[test]
-  fn query_usbip() {
+  #[should_panic]
+  fn query_not_installed() {
     let pacman = Alpm::new().unwrap();
 
-    assert_eq!("4.7-2".to_string(), pacman.query_package_version("usbip").unwrap());
+    pacman.query_package_version("non-existing").unwrap();
```

Comment:
Did you replace the usbip test?

I inserted that because it has a shorter version number.